Sorting.rb

```diff
@@ -0,0 +1,31 @@
+class Sorting
+
+  def self.sort(arr, sort_order)
+    n = arr.length
+    (n-1).times do |i|
+      order(arr, sort_order)
+    end
+    return arr
+  end
+
+  def self.order(arr, sort_order)
```

Comment:
You can make it Private .As we don't need order method to be called by our class or instance, They should only be called from methods .